GoogleSignIn/Tests/Unit/GIDFakeFetcherService.m

```diff
@@ -63,6 +63,15 @@ - (NSURLSession *)sessionForFetcherCreation {
   return nil;
 }
 
+// TODO: replace with GTMSessionFetcherSessionCreationBlock once that has become
+// available by updating the version dependencies.
+typedef NSURLSession *_Nullable (^GIDFakeFetcherSessionCreationBlock)(
```

Comment:
Maybe move the typedef above for readability?

Reply:
The upcoming fetcher changes have been tweaked, the end result being that rather than adding a new method, the fetcher is moving most of the methods out of the protocol; but then the fake fetcher service has some `@synthesize` properties that aren't defined (until the fetcher update gets pushed as a release).